src/OSSupport/Errors.cpp

```diff
@@ -5,11 +5,11 @@
 
 AString GetOSErrorString( int a_ErrNo)
 {
-	char buffer[ 1024 ];
 	AString Out;
 
 	#ifdef _WIN32
 
+	wchar_t buffer[1024];
 	FormatMessage(FORMAT_MESSAGE_FROM_SYSTEM, nullptr, a_ErrNo, 0, buffer, ARRAYCOUNT(buffer), nullptr);
 	Printf(Out, "%d: %s", a_ErrNo, buffer);
```

Comment:
This is broken because %s is for single byte encoding unit strings, not multi-byte encoding unit strings.